packages/FormElement/src/components/Content/Content.js

```diff
@@ -10,24 +12,54 @@ const propTypes = {
   refLabel: RefOf(),
   /** Used for aria-describedby on the FormElement */
   ariaDescribedBy: PropTypes.string,
+  /** Used when form Elements are nested and is automatically appended to aria-describedby */
+  wrapperAriaDescribedBy: PropTypes.string,
 };
 
 const defaultProps = {
   idForLabel: null,
   ariaDescribedBy: null,
   refLabel: null,
+  wrapperAriaDescribedBy: null,
```

Comment:
why do we have ariaDescribedBy and wrapperAriaDescribedBy, shouldn't this be part of a more universal prop named a11yText (this is what we use across our library)? 

Do you have an example where we are using `wrapperAriaDescribedBy`?

Reply:
You can see how we use it in this storybook example - https://github.com/acl-services/paprika/pull/663/files#diff-1a53d8778ef9b8807972d1aa4463c8c9R54

In the case of wrapping FormElements with other FormElements we need to make sure that the `aria-describedBy` strings coming from the wrapping FormElement are being passed down to the children 

Perhaps a better name for these props could be `a11yDescribedByIds` so I'll switch to that